qvm-examples.asd

```diff
@@ -4,10 +4,16 @@
 
 (asdf:defsystem #:qvm-examples
   :description "Examples using the QVM."
-  :author "Robert Smith <robert@rigetti.com>"
+  :author "Robert Smith <robert@rigetti.com>, Juan M. Bello-Rivas <jbellorivas@rigetti.com>"
   :license "Apache License 2.0 (See LICENSE.txt)"
-  :depends-on (#:qvm)
+  :depends-on (
+               ;; Nelder-Mead
+               #:cl-grnm
+               ;; Quantum Virtual Machine
+               #:qvm
```

Comment:
Looks like this should also depend on `qvm-app`.

Reply:
Well spotted. Thank you!